CHANGELOG.md

```diff
@@ -25,6 +25,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `opentelemetry-instrumentation-asyncpg` Fix fallback for empty queries.
   ([#3253](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3253))
 
+### Added
```

Comment:
We have already an Added section above

Reply:
@xrmx will fix now. Thanks. Must have gotten messed up in the rebase